scripts/wdio/clean-screenshots-cli.js

```diff
@@ -8,10 +8,12 @@ const packageJson = require('../../package.json');
 commander
   .version(packageJson.version)
   .option('--removeReference', 'Whether or not to remove the reference screenshots', false)
+  .option('--snapshotDirectory', 'The directory that clean should check for snapshots folders')
```

Comment:
You could set '**' as default value here.
```suggestion
  .option('--snapshotDirectory', 'The directory that clean should check for snapshots folders', '**')
```
This makes some of your tests unnecessarry.

Reply:
Internally we default to **, I intended this to be a folder. If we default to ** we would search in **/** at the end.
The tests are for the function since the clean is invoked from other places other than this cli. The cli seems to be a front-facing tool that allows the clean to be run from the command line. Other consumers are directly invoking the clean function, so we can't rely on option defaulting alone.